app/Actions/Albums/PositionData.php

```diff
@@ -2,37 +2,37 @@
 
 namespace App\Actions\Albums;
 
-use App\Actions\Album\Extensions\LocationData;
-use App\Actions\Albums\Extensions\PublicIds;
+use App\Actions\PhotoAuthorisationProvider;
 use App\Models\Configs;
 use App\Models\Photo;
 
 class PositionData
 {
-	use LocationData;
+	protected PhotoAuthorisationProvider $photoAuthorisationProvider;
+
+	public function __construct()
+	{
+		$this->photoAuthorisationProvider = resolve(PhotoAuthorisationProvider::class);
```

Comment:
We can't use DDI in the `__construct()` ?

Reply:
There is no "can't". Don't know why I did it differently here than in other places. I changed it.